ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1393 @@
+{
```

Comment:
<p>Imports should be at the top</p><p> </p><p><code>imshow</code>  is a function from <code>matplotlib.pyplot</code>,  in general it is bad to re-use names of common functions</p><p> </p><p>If you want to show your data, show one-two images and print corresponding labels. Right now I can see a dozen of barely visible images and a bunch of numbers that make no sense to me.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>reworked</p>